test/Performance/setup_env_unix.sh

```diff
@@ -1,140 +1,184 @@
+    printf "done\n"
+
+    printf "Installing svn..."
+    brew install svn >> env_setup.log
+    printf "done\n"
+
+    printf "Installing openssl..."
+    brew install pkg-config >> env_setup.log
+    brew install openssl >> env_setup.log
+    printf "done\n"
+
+    printf "Installing python3..."
+    brew install python3 >> env_setup.log
+    printf "done\n"
+
+    printf "Installing MSODBCSQL..."
+    brew tap microsoft/msodbcsql https://github.com/Microsoft/homebrew-msodbcsql >> env_setup.log
+    brew update >> env_setup.log
+    yes | ACCEPT_EULA=Y brew install msodbcsql >> env_setup.log
+    printf "done\n"
+
+    yes | brew install autoconf >> env_setup.log
+    printf "Installing pyodbc..."
+    pip3 install pyodbc >> env_setup.log
+    printf "done\n"
 fi
-echo "Downloading PHP-$PHP_VERSION source tarball..."
+
+printf "Downloading PHP-$PHP_VERSION source tarball..."
+#   pipe non-error to log file
 wget http://ca1.php.net/get/php-$PHP_VERSION.tar.gz/from/this/mirror -O php-$PHP_VERSION.tar.gz >> env_setup.log 2>&1
-echo "OK"
-echo "Extracting PHP source tarball..."
+printf "done\n"
+
+printf "Extracting PHP source tarball..."
 rm -rf php-$PHP_VERSION
 tar -xf php-$PHP_VERSION.tar.gz
-echo "OK"
-cd php-$PHP_VERSION
-mkdir ext/sqlsrv ext/pdo_sqlsrv
-cp -r $DRIVER_SOURCE_PATH/sqlsrv/* $DRIVER_SOURCE_PATH/shared ext/sqlsrv
-cp -r $DRIVER_SOURCE_PATH/pdo_sqlsrv/* $DRIVER_SOURCE_PATH/shared ext/pdo_sqlsrv
+printf "done\n"
+
+phpDir=php-$PHP_VERSION
+cd $phpDir
+
+printf "Configuring PHP..."
 ./buildconf --force >> ../env_setup.log 2>&1
-CONFIG_OPTIONS="--enable-cli --enable-cgi --enable-pdo --enable-sqlsrv=shared --with-pdo_sqlsrv=shared --with-odbcver=0x0380 --with-zlib --enable-mbstring --prefix=/usr/local"
+CONFIG_OPTIONS="--enable-cli --enable-cgi --with-zlib --enable-mbstring --prefix=/usr/local"
 [ "${PHP_THREAD}" == "ts" ] && CONFIG_OPTIONS=${CONFIG_OPTIONS}" --enable-maintainer-zts"
 if [ $PLATFORM = "Sierra" ]; then
     CONFIG_OPTIONS=$CONFIG_OPTIONS" --with-openssl=/usr/local/opt/openssl/"
 else
     CONFIG_OPTIONS=$CONFIG_OPTIONS" --with-openssl"
 fi
-echo "Configuring PHP..."
+#pipe non-error to log file
 (./configure $CONFIG_OPTIONS >> ../env_setup.log 2>&1)
-echo "OK"
-echo "Compiling PHP and the drivers..."
-make >> ../env_setup.log 2>&1
-echo "OK"
-sudo make install >> ../env_setup.log 2>&1
+printf "done\n"
+
+printf "Compiling and installing PHP..."
+make >> ../env_setup.log
+sudo make install >> ../env_setup.log
+printf "done\n"
+
+# check PHP version
+/usr/local/bin/php -v
+
+printf "Setting up drivers..."
+phpExtDir=`/usr/local/bin/php-config --extension-dir`
+cp -r $SQLSRV_DRIVER $phpExtDir/sqlsrv.so
+cp -r $PDO_DRIVER $phpExtDir/pdo_sqlsrv.so
 cp php.ini-production php.ini
 echo "extension=sqlsrv.so" >> php.ini
 echo "extension=pdo_sqlsrv.so" >> php.ini
 sudo cp php.ini /usr/local/lib
+printf "done\n"
+
+# check drivers
+/usr/local/bin/php --ri sqlsrv
+/usr/local/bin/php --ri pdo_sqlsrv
```

Comment:
the previous script assumes the current user who runs has that in the path, which is the case in our test lab. The instructions tells you to run "sudo env "PATH=$PATH" bash blah" which passes the path of the current user to sudo because path isn't preserved. I don't think we should assume that.